tartiflette/schema/transformer.py

```diff
@@ -750,6 +767,127 @@ def parse_directive_definition(
     return directive
 
 
+def parse_enum_type_extension(
+    enum_type_extension_node: "EnumTypeExtensionNode", schema: "GraphQLSchema"
+) -> "GraphQLEnumTypeExtension":
+
+    enum_extension = GraphQLEnumTypeExtension(
+        name=parse_name(enum_type_extension_node.name, schema),
+        directives=enum_type_extension_node.directives,
+        values=parse_enum_values_definition(
+            enum_type_extension_node.values, schema
+        ),
+    )
+
+    schema.add_extension(enum_extension)
+
+    return enum_extension
+
+
+def parse_input_object_type_extension(
+    input_object_type_extension_node: "InputObjectTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLInputObjectTypeExtension":
+
+    input_object_extenstion = GraphQLInputObjectTypeExtension(
+        name=parse_name(input_object_type_extension_node.name, schema),
+        directives=input_object_type_extension_node.directives,
+        input_fields=parse_input_fields_definition(
+            input_object_type_extension_node.fields, schema
+        ),
+    )
+
+    schema.add_extension(input_object_extenstion)
+
+    return input_object_extenstion
+
+
+def parse_object_type_extension(
+    object_type_extension_node: "ObjectTypeExtensionNode",
+    schema: "GraphQLSchema",
+) -> "GraphQLObjectTypeExtension":
+
+    object_extension = GraphQLObjectTypeExtension(
+        name=parse_name(object_type_extension_node.name, schema),
+        fields=parse_fields_definition(
+            object_type_extension_node.fields, schema
+        ),
+        directives=object_type_extension_node.directives,
+        interfaces=[
```

Comment:
Shouldn't it be:
```suggestion
        interfaces=parse_implements_interfaces(
            object_type_extension_node.interfaces, schema
        )
```

Reply:
But I only care about the name, then in the bake method of the extension i'll have to do a `[x.name.value for x in self.interfaces]` that's why I took the direct approach.